### PR TITLE
i18n(es): Update `getting-started.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Update `getting-started.mdx` [`0100298`](https://github.com/withastro/starlight/commit/0100298b158f90cf2beed67e9a185f40a533465b)

### CODE REVIEW

File: docs/src/content/docs/es/getting-started.mdx [hunk 1]
```diff
@@ -45,24 +46,68 @@ Prueba Starlight en tu navegador:
...
-### Frontmatter con seguridad de tipos
+</TabItem>
+</Tabs>
+
+Esto imprimirá un mensaje en tu terminal con la URL de la vista previa local.
+Abre esta URL para comenzar a navegar por tu sitio.
+
+### Añade contenido
+
+Starlight está listo para que añadas nuevo contenido o traigas ¡tus archivos existentes!.
```
Comment (Waxer59) [inline docs/src/content/docs/es/getting-started.mdx:84]:
```suggestion
Starlight está listo para que añadas nuevo contenido o ¡traigas tus archivos existentes!.
```

File: docs/src/content/docs/es/getting-started.mdx [hunk 2]
```diff
@@ -45,24 +46,68 @@ Prueba Starlight en tu navegador:
...
+Starlight admite la creación de contenido en Markdown y MDX sin necesidad de configuración. Puedes agregar compatibilidad con Markdoc instalando la [integración experimental de Astro Markdoc](https://docs.astro.build/es/guides/integrations-guide/markdoc/).
+
+#### Añade páginas
+
+Añade nuevas páginas a tu sitio creando archivos `.md` o `.mdx` en `src/content/docs/`.
+Utiliza subcarpetas para organizar tus archivos y crear múltiples segmentos de ruta.
+
+Por ejemplo, la siguiente estructura de archivos generará páginas en `example.com/hello-world` y `example.com/guides/faq`:
+
+import FileTree from '../../components/file-tree.astro';
```
Comment (Waxer59) [inline docs/src/content/docs/es/getting-started.mdx:97]:
```suggestion
import FileTree from '../../../components/file-tree.astro';
```

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/getting-started.mdx [hunk 3]
```diff
@@ -45,24 +46,68 @@ Prueba Starlight en tu navegador:
 [abre la plantilla en StackBlitz](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics).
 :::
 
-## Creando contenido con Starlight
+### Inicia el servidor de desarrollo
 
-Starlight está listo para que añadas nuevo contenido o traigas tus archivos existentes.
+Cuando trabajas localmente, el [servidor de desarrollo de Astro](https://docs.astro.build/en/reference/cli-reference/#astro-dev) te permite previsualizar tu trabajo y actualiza automáticamente tu navegador cuando realizas cambios.
```
Comment (delucis) [inline docs/src/content/docs/es/getting-started.mdx:51]: We should link to the Spanish docs here:

```suggestion
Cuando trabajas localmente, el [servidor de desarrollo de Astro](https://docs.astro.build/es/reference/cli-reference/#astro-dev) te permite previsualizar tu trabajo y actualiza automáticamente tu navegador cuando realizas cambios.
```

File: docs/src/content/docs/es/getting-started.mdx [hunk 4]
```diff
@@ -75,16 +120,18 @@ description: Esta es una página en mi sitio impulsado por Starlight
...
-## Desplegando tu sitio web de Starlight
+### Siguientes pasos
 
-Una vez que hayas creado y personalizado tu sitio web de Starlight, puedes implementarlo en un servidor web o plataforma de alojamiento de tu elección, incluyendo Netlify, Vercel, GitHub Pages y muchos más.
-
-[Aprende sobre cómo desplegar un sitio Astro en la documentación de Astro.](https://docs.astro.build/es/guides/deploy/)
+- **Configura:** Aprende acerca de las opciones comunes en [“Personalizando Starlight”](/guías/personalización/).
+- **Navega:** Configura tu barra lateral siguiendo la guía [“Barra Lateral de Navegación”](/guides/sidebar/).
+- **Componentes:** Descubre tarjetas incorporadas, pestañas y más en la guía [“Componentes”](/guides/components/).
+- **Despliega:** Publica tu trabajo siguiendo la guía [“Despliega tu sitio”](https://docs.astro.build/en/guides/deploy/) en la documentación de Astro.
```
Comment (delucis) [inline docs/src/content/docs/es/getting-started.mdx:128]: Same here:
```suggestion
- **Despliega:** Publica tu trabajo siguiendo la guía [“Despliega tu sitio”](https://docs.astro.build/es/guides/deploy/) en la documentación de Astro.
```

File: docs/src/content/docs/es/getting-started.mdx [hunk 5]
```diff
@@ -75,16 +120,18 @@ description: Esta es una página en mi sitio impulsado por Starlight
...
 
-## Desplegando tu sitio web de Starlight
+### Siguientes pasos
 
-Una vez que hayas creado y personalizado tu sitio web de Starlight, puedes implementarlo en un servidor web o plataforma de alojamiento de tu elección, incluyendo Netlify, Vercel, GitHub Pages y muchos más.
-
-[Aprende sobre cómo desplegar un sitio Astro en la documentación de Astro.](https://docs.astro.build/es/guides/deploy/)
+- **Configura:** Aprende acerca de las opciones comunes en [“Personalizando Starlight”](/guías/personalización/).
+- **Navega:** Configura tu barra lateral siguiendo la guía [“Barra Lateral de Navegación”](/guides/sidebar/).
+- **Componentes:** Descubre tarjetas incorporadas, pestañas y más en la guía [“Componentes”](/guides/components/).
```
Comment (delucis) [inline docs/src/content/docs/es/getting-started.mdx:127]: Looks like these links need fixing:

```suggestion
- **Configura:** Aprende acerca de las opciones comunes en [“Personalizando Starlight”](/es/guides/customization/).
- **Navega:** Configura tu barra lateral siguiendo la guía [“Barra Lateral de Navegación”](/es/guides/sidebar/).
- **Componentes:** Descubre tarjetas incorporadas, pestañas y más en la guía [“Componentes”](/es/guides/components/).
```

File: docs/src/content/docs/es/getting-started.mdx [hunk 6]
```diff
@@ -93,37 +140,34 @@ Starlight es una integración Astro y se actualiza de la misma manera que cualqu
...
 
-Puedes ver una lista completa de los cambios realizados en cada versión en [el registro de cambios de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md).
+Consulta el [registro de cambios de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) para obtener una lista completa de los cambios realizados en cada versión.
 
 ## Solución de problemas de Starlight
 
-Tanto la [configuración del proyecto](/es/reference/configuration/) como la [configuración individual de frontmatter](/es/reference/frontmatter/) de cada página están disponibles en la sección de Referencia de este sitio. Utiliza estas páginas para asegurarte de que tu sitio Starlight está configurado y funcionando correctamente.
-
-Revisa la creciente lista de guías en la barra lateral para obtener ayuda para agregar contenido y personalizar tu sitio Starlight.
+Utiliza las páginas de referencia de [configuración del proyecto](/reference/configuration/) y [configuración del frontmatter de páginas individuales](/reference/frontmatter/) para asegurarte de que tu sitio de Starlight esté configurado y funcione correctamente.
```
Comment (delucis) [inline docs/src/content/docs/es/getting-started.mdx:167]: Same here:

```suggestion
Utiliza las páginas de referencia de [configuración del proyecto](/es/reference/configuration/) y [configuración del frontmatter de páginas individuales](/es/reference/frontmatter/) para asegurarte de que tu sitio de Starlight esté configurado y funcione correctamente. 
```

File: docs/src/content/docs/es/getting-started.mdx [hunk 7]
```diff
@@ -93,37 +140,34 @@ Starlight es una integración Astro y se actualiza de la misma manera que cualqu
...
-
-Revisa la creciente lista de guías en la barra lateral para obtener ayuda para agregar contenido y personalizar tu sitio Starlight.
+Utiliza las páginas de referencia de [configuración del proyecto](/reference/configuration/) y [configuración del frontmatter de páginas individuales](/reference/frontmatter/) para asegurarte de que tu sitio de Starlight esté configurado y funcione correctamente. 
+Consulta las guías en la barra lateral para obtener ayuda en la adición de contenido y personalización de tu sitio de Starlight.
 
-Si no encuentras la respuesta en la documentación, visita la [documentación completa de Astro](https://docs.astro.build) para obtener la documentación completa de Astro. Es posible que tu pregunta se responda comprendiendo cómo funciona Astro en general, debajo de este tema Starlight.
+Si no encuentras la respuesta en la documentación, visita la [documentación completa de Astro](https://docs.astro.build) para obtener la documentación completa de Astro.
```
Comment (delucis) [inline docs/src/content/docs/es/getting-started.mdx:170]: And it probably also makes sense here to link to the Spanish version of the Astro Docs:

```suggestion
Si no encuentras la respuesta en la documentación, visita la [documentación completa de Astro](https://docs.astro.build/es/) para obtener la documentación completa de Astro.
```

File: docs/src/content/docs/es/getting-started.mdx [hunk 8]
```diff
@@ -5,33 +5,34 @@ description: Aprende cómo empezar a construir tu próximo sitio de documentaci
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
-## Creando un nuevo proyecto
-
 Starlight es un tema de documentación completo construido sobre el framework [Astro](https://astro.build).
+Esta guía te ayudará a comenzar con un nuevo proyecto. 
+Consulta las [instrucciones de configuración manual](/manual-setup/) para agregar Starlight a un proyecto de Astro existente.
```
Comment (delucis) [inline docs/src/content/docs/es/getting-started.mdx:10]: Last one!

```suggestion
Consulta las [instrucciones de configuración manual](/es/manual-setup/) para agregar Starlight a un proyecto de Astro existente.
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thank you for the update @Waxer59 — and thanks for reviewing @dreyfus92!

Just caught a few links we need to fix.

**Comment by Waxer59** [comment]
Thanks for the review 🤠!,

**Review by kevinzunigacuellar — APPROVED** [review]
Let's Get This Moving 😆

**Review by delucis — APPROVED** [review]
Nice work team 🙌

**Pull request merged**